spec/features/home_with_new_permissions_spec.rb

```diff
@@ -0,0 +1,146 @@
+require "rails_helper"
+
+RSpec.feature "Home page with new permissions", type: :feature do
```

Comment:
Duplicated `spec/features/admin/facilities/index_with_new_permissions_spec.rb`, but perform the new permission checks instead. This feature spec covers both `admin/facilities_controller` and `admin/facility_groups_controller`